src/test/java/de/uni_hamburg/informatik/swt/se2/mediathek/services/vormerk/VormerkServiceImplTest.java

```diff
@@ -64,9 +64,8 @@ public VormerkServiceImplTest()
         medienbestand.fuegeMediumEin(_medium);
         medienbestand.fuegeMediumEin(_medium2);
         
-        _verleihService = new VerleihServiceImpl(kundenstamm, medienbestand, new ArrayList<Verleihkarte>());
-        
-        
+        _vormerkService = new VormerkServiceImpl(null);//temporär null
```

Comment:
Das wird so nicht klappen, in Zeile 59 gibst du die momentane Referenz an VerleihServiceImpl weiter, jedoch setzt du in Zeile _vormerkService auf eine neue Instanz, _service zeigt jedoch noch auf die alte!
Das ist so wie
x = new FalschesX()
y = new Y(x)
x = new RichtigesX()

y zeigt immer noch auf das falsche X!